biggest_rand_number.rb

```diff
@@ -0,0 +1,14 @@
+rand_array = []
+
+10.times do |index|
+  rand_array.push(rand(0..100))
+end
```

Comment:
Poderia fazer melhor, escovando bits, jaja

```suggestion
numbers = (1..10).to_a.map do |number|
  rand(0..100)
end
```
Assim, você terá numbers recebendo o valor do map, (map sempre retorna um array do último valor da lógica dentro dele, `each` não retorna nada, importante saber essa diferença, então há o range (1...10) que transformamos isso em array com o `to_a`.

O que pensa sobre essa solução?

Reply:
[DÚVIDA]

Muito bom, mas o .map já gera um novo array, então seria necessário usar o .ta_a?